### DISCUSSION TITLE
CRUD - replace :id with a different var

### DISCUSSION THREAD

**Discussion by aba-koepke** [mockoon/mockoon#1038] [Help] [answered]
Hi there, i like to ask if its possible to use a different var instead of "id" in a databucket to do CRUD-Operations, we normally name unique-Ids "uuid" in our datasets..

Thanks for any Help.

**Comment by 255kb** [comment, marked as answer]
Hi, it's currently not possible but it's an option I would like to add. Could you please open an issue? 
Thank you :)

**Reply by aba-koepke** [reply]
Thanks. I openend an Issue.
